### ISSUE TITLE
BUG - Forms component is broken

### ISSUE THREAD

**Issue by khawkins98** [visual-framework/vf-core#489]
Looks like something has broken on `vf-form`

https://dev.beta.embl.org/guidelines/visual-framework/dev-docs/components/detail/vf-form__core.html

## 1. The label floats, well, don't:

![image](https://user-images.githubusercontent.com/928100/61369690-f58e2000-a890-11e9-9a85-6b2ecf03bd7b.png)

The [float-labels.js](https://pryley.github.io/float-labels.js/) seems to still be there, but is either not being invoked and/or needs some css to control the display. 

Not sure if there's been much progress on CSS-only approaches: https://medium.com/@eugeniolujambio/accessible-css-only-floating-labels-74bf9173bc5f ... if so, IE 11 could probably live with a fallback approach.

## 2. The radio buttons seem to have broke:

![image](https://user-images.githubusercontent.com/928100/61369778-266e5500-a891-11e9-813d-66edd9967c34.png)

**Comment by khawkins98** [comment]
On the radio buttons, looks like it lost settings for border width/style; adding this back seems to fix it:
```css
.vf-form__radio + .vf-form__label::before {
    border-width: 3px;
    border-style: solid;
}
```

![image](https://user-images.githubusercontent.com/928100/61370407-a34dfe80-a892-11e9-8551-ada227bf0991.png)

**Comment by sturobson** [comment]
radio buttons are now fixed in #497 

todo: the JS interaction of things for the input slide up whizz bang
